tests/prefetch_related/tests.py

```diff
@@ -1161,3 +1161,24 @@ def test_bug(self):
         prefetcher = get_prefetcher(self.rooms[0], 'house')[0]
         queryset = prefetcher.get_prefetch_queryset(list(Room.objects.all()))[0]
         self.assertNotIn(' JOIN ', force_text(queryset.query))
+
+
+class Ticket24873Tests(TestCase):
```

Comment:
I think it is not good idea to add more testcases like `Ticket0000Tests`.
This bug is not epic well-known issue, so maybe this test should be placed near all prefetch tests with the name like `test_nested_prefetch_related_are_not_overwritten`.